src/components/Article.jsx

```diff
@@ -11,8 +12,17 @@ import Trending from 'components/Trending';
 
 export default class Article extends BaseComponent {
   render () {
+    const meta = [
+      {property: "og:title", content: this.props.title},
+      {property: "og:type", content: "article"},
+      {property: "og:url", content: this.props.url},
+      {property: "og:image", content: this.props.featuredImage},
+      {property: "og:description", content: this.props.teaser},
+      {property: "og:site_name", content: "The Gazelle"},
+    ];
     return (
       <div className="article">
+        <DocMeta tags={meta} />
```

Comment:
`DocMeta` adds the meta tags directly into the header. It works for nested components too. It's a dependency I'm using until we get Ghost up and running which should add the OG tags automatically. Without them, sharing articles from the summer issue could look weird.